src/env.d/08-ca-certificates

```diff
@@ -0,0 +1,3 @@
+# Set the ssl certificates path for ruby.
+export SSL_CERT_DIR="${CREW_PREFIX}/etc/ssl/certs"
+export SSL_CERT_FILE"${CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt
```

Comment:
Please change to:
```
SSL_CERT_DIR="${CREW_PREFIX}/etc/ssl/certs"
SSL_CERT_FILE"${CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt
```
`CREW_PREFIX` is set in /usr/local/etc/profile, so we will always have that as an env variable.

Reply:
(Note the quotation mark at the end of the second line.)

Reply:
Now just this:

```
SSL_CERT_FILE="${CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt
```